pyproject.toml

```diff
@@ -25,7 +25,7 @@ license = {"file" = "LICENSE"}
 keywords = ["cobald", "tardis", "High Energy Physics"]
 requires-python = "~=3.6"
 dependencies = [
-    "cobald",
+    "cobald~=0.12",
```

Comment:
We might think about leaving some instructions in our documentation, that the version of this repo is bumped with each release of cobald to ensure, that everything is working as expected. New issue?

Reply:
Opened #3 to track this.